server/static/dashboard/collab.js

```diff
@@ -7,42 +10,486 @@ collab.fetch(function(err) {
   document.querySelector('#partners').textContent = collab.data.users.slice().sort().join(' & ')
 });
 
+// Parameters required to use updateFunction
+// TODO: Reconcile parameters and extraArgs
+var parameters = {};
+
+// Whether deleted code is currently shown or not
+var showDeletedCode = false;
```

Comment:
Let's have a single piece of global view state for all options (deleted code, regexes, anything else?) that is either: global obj updated by event handlers, or global DOM nodes inspected by updaters.

Reply:
Used a global object.